src/main/resources/META-INF/mods.toml

```diff
@@ -21,7 +21,7 @@ issueTrackerURL="https://github.com/Ridanisaurus/Emendatus-Enigmatica/issues" #o
 modId="emendatusenigmatica" #mandatory
 
 # The version number of the mod - there's a few well known ${} variables useable here or just hardcode it
-version="1.16.4-1.1.3" #mandatory
+version="1.16.4-1.1.4" #mandatory
```

Comment:
This line and the two lines in `Strata.java` are the only real changes.
The rest is all from generated data.